### DISCUSSION TITLE
buddhism/2021/koans/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#61] [General]
# buddhism/2021/koans/

https://shanenull.com/buddhism/2021/koans/

**Comment by shane0** [comment]
The "Shit Stick" koan, attributed to Zen Master Unmon, is known for its unconventional and provocative nature, typical of many Zen teachings. Different versions of the koan exist, and it's often used as a tool for meditation and contemplation in Zen practice. Here are a few additional aspects and interpretations related to this koan:

1. **Beyond Intellectual Understanding:** The koan challenges the student to move beyond conceptual thinking and intellectual understanding. The use of a seemingly crude or mundane object, like a "shit stick," is meant to push the practitioner to a direct, experiential realization.

2. **Breaking Dualistic Thinking:** The koan challenges dualistic thinking, where we categorize things into 'clean' and 'dirty,' 'sacred' and 'profane.' By using an everyday object associated with impurity, Unmon encourages the student to transcend such dualities and see the inherent purity in all things.

3. **Direct Experience:** The koan aims to lead the practitioner to an immediate, non-conceptual experience of reality. Instead of relying on words and concepts, the student is encouraged to grasp the truth directly.

4. **Iconoclasm:** Zen teachings often include elements of iconoclasm, breaking down conventional or revered symbols to point directly to the essence of reality. In this case, using a "shit stick" challenges conventional notions of purity and sacredness.

5. **Variations in Responses:** There are variations in the recorded responses to this koan, emphasizing different aspects of the teaching. In some versions, Unmon hits the monk with a stick, emphasizing the immediacy of direct experience over verbal explanations.

6. **The Oak Tree:** In some versions, after the question about Buddha, Unmon responds with "The oak tree in the front garden." This part of the koan introduces another seemingly ordinary element, inviting the student to see the profound in the everyday.

It's important to note that the essence of a koan is often in the practitioner's personal, experiential realization rather than in a rational understanding. Different teachers and traditions may emphasize different aspects of the koan, and students might explore it over an extended period as part of their Zen practice.
